test/utils/config.ts

```diff
@@ -1,31 +1,33 @@
-import nock from 'nock'
-const fs = require('fs')
-const path = require('path')
+import nock from "nock";
+const fs = require("fs");
```

Comment:
Codacy found an issue: [require statement not part of an import statement](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6083797)